doc/manual/src/introduction.md

```diff
@@ -1,193 +1,144 @@
-
-Once a package is built, runtime dependencies are found by scanning
-binaries for the hash parts of Nix store paths (such as `r8vvq9kq…`).
-This sounds risky, but it works extremely well.
+# Welcome to Nix
+
+# Where Nix comes from
+
+Nix was created by Eelco Dolstra and developed as the subject of his PhD thesis [The Purely Functional Software Deployment Model](https://edolstra.github.io/pubs/phd-thesis.pdf), published 2006.
+Over the years, many others have joined the effort to make software work reliably, now and in the long run.
+Today, a [world-wide developer community](https://nixos.org/) contributes to Nix and the ecosystem that has grown around it.
+
+# How Nix works
+
+> The name *Nix* is derived from the Dutch word *niks*, meaning *nothing*;
+> build actions do not see anything that has not been explicitly declared as an input.
+>
+> — [Nix: A Safe and Policy-Free System for Software Deployment](https://edolstra.github.io/pubs/nspfssd-lisa2004-final.pdf), LISA XVIII, 2004
+
+Nix primarily deals with making sure that files are complete and available when processes need them.
+This is accomplished by two measures:
+- All files and their relations are specified by cryptographic hash in the Nix store, and are made immutable.
+  Files are only deleted when nothing refers to them any more.
+  This guarantees that a successful program run can be repeated, because nothing will change or disappear inadvertently.
+- Processes are executed in a clean environment, where they can only access what is explictly allowed.
+  This prevents obtaining useful results without specifyig all dependencies.
+
+Nix has become known as the *purely functional package manager*, because this approach to ensure repeatability allows escaping from [dependency hell](https://en.wikipedia.org/wiki/Dependency_hell) and taming the complexity of the software ecosystem.
+It is grown an ecosystem of exceptionally powerful tools – including Nixpkgs, , and NixOS, a Linux distribution that can be configured fully declaratively, with unmatched flexibility.
+
+At the heart of this is the [Nix store](./store/index.md).
+It keeps immutable file system objects, and tracks dependencies between them.
+It can also execute sandboxed processes and efficiently copy bundled dependencies across machines in a way that preverves correctness.
+
+Nix comes with a purely functional programming language to describe and compose these computations, and succinctly refer to their inputs and outputs: the [Nix language](./language/index.html).
+
+# What Nix can do
+
+Nix enforces *complete, exact* dependencies whenever software under its control is run.
+This makes possible unique features without compromising on efficiency.
+
+## Multiple software versions side by side
+
+You can have multiple versions or variants of a package installed at the same time.
+Different files always end up on different paths in the Nix store, so they don’t interfere with each other.
+This circumvents the “DLL hell”.
```

Comment:
```suggestion
This circumvents the “Dependency hell”.
```

https://en.m.wikipedia.org/wiki/Dependency_hell

We do not have good windows support yet to circumvent a DLL hell yet.